src/main/java/games/strategy/engine/framework/startup/ui/SetupPanel.java

```diff
@@ -66,8 +71,8 @@ public void preStartGame() {}
   public void postStartGame() {}
 
   @Override
-  public ILauncher getLauncher() {
-    throw new IllegalStateException("NOt implemented");
+  public GameLauncher getLauncher() {
+    throw new IllegalStateException("Not implemented");
```

Comment:
Instead of throwing an exception, why not marking this method abstract? the class is already abstract.

Reply:
Good idea.  Or even better, just remove it since it's inherited from the `ISetupPanel` interface.

Unfortunately, it appears that neither the `ClientSetupPanel` nor `MetaSetupPanel` subclasses implement this method, so either marking it `abstract` or removing it will introduce two compiler errors.

If throwing an exception is retained, it might be more self-documenting to throw either `UnsupportedOperationException` or `org.apache.commons.lang3.NotImplementedException`.

Reply:
Regret to say I'd rather not fix this one here (scope creep). The only intended update here is a typo fix, and ILauncher and so that had to be updated.